### DISCUSSION TITLE
docker版怎样设置才能支持https?

### DISCUSSION THREAD

**Discussion by lyh458** [messense/aliyundrive-webdav#380] [Q&A] [answered]
大家好：

因为ios现在不支持http，只能https。而使用[aliyundrive-webdav](https://github.com/messense/aliyundrive-webdav)的docker版，按照README中的指令默认是http访问，请问有没有像snap/pip版中配置tls证书的option，从而实现https访问？

非常感谢！

**Comment by messense** [comment, marked as answer]
```bash
docker run -d --name=aliyundrive-webdav --restart=unless-stopped \
  -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
  -p 8080:8080 \
  -e REFRESH_TOKEN='your refresh token' \
  -e WEBDAV_AUTH_USER=admin \
  -e WEBDAV_AUTH_PASSWORD=admin \
  -e TLS_CERT=/etc/aliyundrive-webdav/tls.cert \
  -e TLS_KEY=/etc/aliyundrive-webdav/tls.key \
  messense/aliyundrive-webdav
```

把证书和私钥文件放到 `/etc/aliyundrive-webdav` 目录中。

**Reply by lyh458** [reply]
> ```shell
> docker run -d --name=aliyundrive-webdav --restart=unless-stopped \
>   -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
>   -p 8080:8080 \
>   -e REFRESH_TOKEN='your refresh token' \
>   -e WEBDAV_AUTH_USER=admin \
>   -e WEBDAV_AUTH_PASSWORD=admin \
>   -e TLS_CERT=/etc/aliyundrive-webdav/tls.cert \
>   -e TLS_KEY=/etc/aliyundrive-webdav/tls.key \
>   messense/aliyundrive-webdav
> ```
> 
> 把证书和私钥文件放到 `/etc/aliyundrive-webdav` 目录中。

请问你是用什么生成证书和私钥文件的？我使用acme生成，偶尔打不开；偶尔能打开，打开后还是提示不安全。

**Comment by Twilight-9** [comment]
这样的https似乎在IOS端无法连接，是否是证书的问题？
